backend/api/Services/Models/IDAInspectionDataResponse.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Text.Json.Serialization;
+#nullable disable
```

Comment:
I do not think we need the #nullable disable

Add required to the fields instead